google/resource_container_cluster.go

```diff
@@ -594,26 +609,31 @@ func resourceContainerClusterCreate(d *schema.ResourceData, meta interface{}) er
 		}
 		op, err = config.clientContainer.Projects.Zones.Clusters.Create(
 			project, zoneName, reqV1).Do()
-		if err != nil {
-			return err
-		}
 	case v1beta1:
 		reqV1Beta := &containerBeta.CreateClusterRequest{}
 		err = Convert(req, reqV1Beta)
 		if err != nil {
 			return err
 		}
-		op, err = config.clientContainerBeta.Projects.Zones.Clusters.Create(
-			project, zoneName, reqV1Beta).Do()
-		if err != nil {
-			return err
+
+		if isRegionalCluster {
```

Comment:
Since the location could also be the zone, I don't think we actually need this conditional- it should be fine to just always do the `Projects.Locations` call

Reply:
That made sense. But didn't what that change to have unknown side-effects. I will give that a try.

Reply:
Fixed.